src/components/SortControl/SortControl.js

```diff
@@ -0,0 +1,57 @@
+import styles      from './SortControl.module.css'
+import {Grid}      from "@mui/material"
+import MenuItem    from '@mui/material/MenuItem';
+import Select      from '@mui/material/Select';
+import FormControl from '@mui/material/FormControl';
+import {useState}  from "react";
+import PropTypes   from "prop-types";
+
+export const RELEASE_DATE = 'Release Date'
+export const TITLE        = 'Title'
+
+const SortControl = ({value, onChange}) => {
+    const [selectedItem, setSelectedItem] = useState(value)
+    const handleChange                    = (e) => {

```

Comment:
too many gaps, would be nice to add eslint in the project
```suggestion
    const handleChange = (e) => {

```

Reply:
it's my WebStorm code formater tried to align code to '='